.travis.yml

```diff
@@ -77,6 +72,10 @@ jobs:
         - cmake --build ringmesh --target package
         - cd $TRAVIS_BUILD_DIR
         - sonar-scanner
+      after_failure:
+        - if [ "$TRAVIS_BRANCH" = "master" ]; then 
```

Comment:
I thingk this will be triggered for PR build also

Reply:
Wait and see